Java/simple-signing-demo/src/main/java/co/upvest/client/SigningUtil.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © 2024 Upvest GmbH <support@upvest.co>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package co.upvest.client;
+
+import org.bouncycastle.asn1.pkcs.PrivateKeyInfo;
```

Comment:
FWIW, @lutzh - minimal research suggests that nobody will have any issue depending on BouncyCastle.